sympy/utilities/lambdify.py

```diff
@@ -179,7 +179,7 @@ def _import(module, reload=False):
 
 
 @doctest_depends_on(modules=('numpy', 'scipy', 'tensorflow',), python_version=(3,))
-def lambdify(args: tUnion[Iterable, 'sympy.core.expr.Expr'], expr: 'sympy.core.expr.Expr', modules=None, printer=None, use_imps=True,
```

Comment:
I'm curious, does anyone know why `'sympy.core.expr.Expr` was used here instead of just `Expr`? Was there a technical reason that the latter did not work? Using both a string and a fully qualified name seems unnecessarily verbose.

Reply:
It was introduced here https://github.com/sympy/sympy/pull/22446 with a single commit with no explanation.